opentracing-api/pom.xml

```diff
@@ -25,8 +25,8 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>opentracing</artifactId>
-    <name>OpenTracing</name>
+    <artifactId>opentracing-api</artifactId>
```

Comment:
yes, but only in their dependencies management file, the package name remains `opentracing` for the API classes. Having this artifact name allows other artifacts downloadable separately, like `opentracing-basictracer`.